provider/datagen/src/tests/make_testdata.rs

```diff
@@ -137,15 +149,15 @@ impl DataExporter for PostcardTestingExporter {
 
         if deallocated != allocated {
             if !EXPECTED_VIOLATIONS.contains(&key) {
-                log::warn!("Zerocopy violation {key} {locale}: {allocated}B allocated, {deallocated}B deallocated");
+                eprintln!("Zerocopy violation {key} {locale}: {allocated}B allocated, {deallocated}B deallocated");
```

Comment:
Question: why did you change this from log::warn to eprintln? log::warn is better because it tells you the filename and line number and you can configure where the log messages go.

Reply:
Ok, makes sense if we're in a test